applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -34,7 +34,6 @@ ZBUS_CHAN_DEFINE(le_audio_chan, struct le_audio_msg, NULL, NULL, ZBUS_OBSERVERS_
 #define BLE_ISO_LATENCY_MS  10
 #define BLE_ISO_RETRANSMITS 2
 
-#if (CONFIG_BT_AUDIO_TX)
```

Comment:
What is the reason for removing this check?

Reply:
Also relevant other places in this file.
Is it to reduce the amount of #if ?

Reply:
Yes, they weren't needed